web/profiles/joinup/config/install/views.view.collections.yml

```diff
@@ -0,0 +1,452 @@
+langcode: en
```

Comment:
Hmm if I am not mistaking, this was my unintentional doing. I wanted to just rename the file from views.view.collection to views.view.collections due to our conversation on whether the title should be collection or collections. But the 2205 had already passed I think (or I didn't push this change). But now there are two files (one collection and one collections) and one must be deleted because the one overrides the other.